pkg/asset/manifests/network.go

```diff
@@ -20,35 +22,18 @@ var (
 	noCfgFilename = filepath.Join(manifestDir, "cluster-network-02-config.yml")
 )
 
-const (
-
-	// We need to manually create our CRD first, so we can create the
-	// configuration instance of it.
-	// Other operators have their CRD created by the CVO, but we manually
-	// create our operator's configuration in the installer.
-	netConfigCRD = `
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: networkconfigs.networkoperator.openshift.io
-spec:
-  group: networkoperator.openshift.io
-  names:
-    kind: NetworkConfig
-    listKind: NetworkConfigList
-    plural: networkconfigs
-    singular: networkconfig
-  scope: Cluster
-  versions:
-    - name: v1
-      served: true
-      storage: true
-`
-)
+// We need to manually create our CRDs first, so we can create the
+// configuration instance of it in the installer. Other operators have
+// their CRD created by the CVO, but we need to create the corresponding
+// CRs in the installer, so we need the CRD to be there.
+// The first CRD is the high-level Network.config.openshift.io object,
+// which is stable ahd minimal. Administrators can override configure the
+// network in a more detailed manner with the operator-specific CR, which
+// also needs to be done before the installer is run, so we provide both.
 
 // Networking generates the cluster-network-*.yml files.
 type Networking struct {
-	config   *netopv1.NetworkConfig
+	config   *configv1.Network
```

Comment:
This field will be made public in #890. You can make it public now or wait for that PR.